examples/react-router/app/components/lists/show.jsx

```diff
@@ -0,0 +1,61 @@
+import Connect from '../../../../../src/addons/connect'
+import ItemForm from './parts/item-form'
+import Items, { addItem, removeItem } from '../../resources/items'
```

Comment:
it was unclear at first that `Items` is the microcosm store and `addItem, removeItem` are actions. I think I'd favor:

```
import Items from '../../resources/items/store'
import { addItem, removeItem } from '../../resources/items/actions'
```

See note below on the index file.